include/boost/compute/algorithm/random_shuffle.hpp

```diff
@@ -46,7 +50,12 @@ inline void random_shuffle(Iterator first,
     // generate shuffled indices on the host
     std::vector<cl_uint> random_indices(count);
     boost::iota(random_indices, 0);
+#ifdef BOOST_COMPUTE_USE_CPP11
+    std::default_random_engine random_engine;
```

Comment:
Would it be better to seed the random engine with `std::random_device`?

Reply:
Yes.  Will fix.